src/index.html

```diff
@@ -17,6 +17,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <a
+        href="#"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Logo"
+        />
+      </a>
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#"
+              class="nav__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li>
+            <a
+              href="#"
+              class="nav__link"
+            >
+              Samsung
+            </a>
+          </li>
```

Comment:
```suggestion
          <li>
            <a
              href="#"
              class="nav__link is-active"
            >
              Apple
            </a>
          </li>
          
          <li>
            <a
              href="#"
              class="nav__link"
            >
              Samsung
            </a>
          </li>
```
Add an empty line between multiple sibling blocks of html